### ISSUE TITLE
No search?

### ISSUE THREAD

**Issue by francisdbillones** [LukeSmithxyz/based.cooking#736]
Search is the one crucial feature that this site is missing IMO. I understand that you can just CTRL + F, but a search bar is more accessible to everyone, and it would allow searching against the content in each recipe as well, using regular old string matching and some simple document processing techniques such as TF/IDF. Since there's no backend this can easily be done on the client side.

**Comment by tfasano1** [comment]
I suggest implementing this and making a pull request, even if it never gets merged other people can choose to use it for their own mirror of based.cooking. Just asking for a feature will usually be a waste of everyone's time.

**Comment by v0iden** [comment]
I agree. Search would be nice. I have no idea how to implement that, though. Would contribute if I could.

**Comment by tfasano1** [comment]
I found this if anyone wants to frankenstein something together: 
https://github.com/akrylysov/simplefts.git
